core/src/main/scala/org/apache/spark/h2o/H2OContext.scala

```diff
@@ -49,18 +50,21 @@ import scala.util.control.NoStackTrace
   * Cluster mode can be set using the spark configuration
   * property spark.ext.h2o.mode which can be set in script starting sparkling-water or
   * can be set in H2O configuration class H2OConf
- */
+  */
 
 /**
   * Create new H2OContext based on provided H2O configuration
   *
   * @param sparkContext Spark Context
   * @param conf H2O configuration
   */
-class H2OContext private (@transient val sparkContext: SparkContext, @transient conf: H2OConf) extends org.apache.spark.Logging
+class H2OContext private (@transient val sparkContext: SparkContext, @transient conf: H2OConf, sqlc: SQLContext) extends org.apache.spark.Logging
```

Comment:
Hmm, conflicting signals I hear... I was convinced that anyway it's a
singleton, and can be retrieved from static.

Thanks,
-Vlad

On Thu, Aug 25, 2016 at 10:23 AM, Michal Malohlava <notifications@github.com
> wrote:

> In core/src/main/scala/org/apache/spark/h2o/H2OContext.scala
> <https://github.com/h2oai/sparkling-water/pull/58#discussion_r76287377>:
>
> >
> >  /**
> >    * Create new H2OContext based on provided H2O configuration
> >    *
> >    * @param sparkContext Spark Context
> >    * @param conf H2O configuration
> >    */
> > -class H2OContext private (@transient val sparkContext: SparkContext, @transient conf: H2OConf) extends org.apache.spark.Logging
> > +class H2OContext private (@transient val sparkContext: SparkContext, @transient conf: H2OConf, sqlc: SQLContext) extends org.apache.spark.Logging
>
> What about implicit parameter? We are already using that in asDataFrame
> calls.
> Do we need SQLContext directly inside H2OContext?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/h2oai/sparkling-water/pull/58/files/dda8bacb29caf9d9ad724922e0bdf29e831b7ecb#r76287377>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAOuKxv4n9iv-bnVp7VjX7b5mI9hNuHLks5qjc-sgaJpZM4Jsgi_>
> .
>